CMakeLists.txt

```
@@ -41,7 +41,7 @@ endmacro ()
 SET_CGNS_BUILD_TYPE()
 
 project("cgns" C)
-
+include(GNUInstallDirs)
```

If possible add a check on UNIX:

```suggestion
if(UNIX)
  include(GNUInstallDirs)
else()
  if (WIN32)
    set(${CMAKE_INSTALL_LIBDIR} "lib")
    set(${CMAKE_INSTALL_DATADIR} "share")
    set(${CMAKE_INSTALL_INCLUDEDIR} "include")
    set(${CMAKE_INSTALL_BINDIR} "bin")
    message(STATUS "Setting installation destination on Windows to: ${CMAKE_INSTALL_PREFIX}")
  else()
    message(FATAL_ERROR "System not UNIX nor WIN32 - not implemented yet")
  endif()
endif()
```